src/utils/walletConnect.ts

```diff
@@ -196,20 +199,24 @@ export function onSessionProposal(
 
   const { chains } = requiredNamespaces.eip155;
   const chainId = parseInt(chains[0].split('eip155:')[1]);
+  const chainIds = chains.map(chain => parseInt(chain.split('eip155:')[1]));
   const peerMeta = proposer.metadata;
-  const dappName = dappNameOverride(peerMeta.name) || 'Unknown Dapp';
+  const dappName =
+    dappNameOverride(peerMeta.url) || peerMeta.name || 'Unknown Dapp';
```

Comment:
is this a diff in v1 vs v2? moving from .name to .url for our overrides? should we not do the same for the same for v1?

Reply:
Nah, the diff here is I mistakenly used `peerMeta.name` in my earlier PRs. `dappNameOverride` expects the URL.